GabrielaRamirez/EjerciciosHackerrank/src/basics/InheritanceII.java

```diff
@@ -0,0 +1,29 @@
+package basics;
+
+    class Arithmetic{
+
+        int add(int x, int y){
+            int c = x + y;
+            return c;
+        }
+
+    }
+
+    class Adder extends Arithmetic{
```

Comment:
According to Java standards and common practices, we should declare every class in its own source file.

https://dzone.com/articles/why-single-java-source-file-can-not-have-more-than

Reply:
i'll put them on their own source